Source/r_segs.c

```diff
@@ -83,6 +83,101 @@ static int64_t  bottomfrac; // [FG] 64-bit integer math
 static fixed_t  bottomstep;
 static int    *maskedtexturecol; // [FG] 32-bit integer math
 
+// WiggleFix: add this code block near the top of r_segs.c
```

Comment:
Could you move this down a bit, right before `R_RenderSegLoop()` so you could use the `HEIGHTBITS` and `HEIGHTUNIT` macros for initialization of the `heightbits` and `heightunit` variables?